index.html

```diff
@@ -305,10 +305,7 @@ <h3>What is a Verifiable Credential?</h3>
 refers to the characteristic of a <a>credential</a> or <a>presentation</a>
 as being able to be <a>verified</a> by a <a>verifier</a>,
 as defined in this document. Verifiability of a credential does not imply
-that the truth of <a>claims</a> encoded therein can be evaluated; however,
-the issuer can include values in the <a>evidence</a> property to help the verifier
-apply their business logic to determine whether the claims have sufficient
-veracity for their needs.
+that truth of <a>claims</a> encoded therein. Rather, once the authenticity and currency of a Verifiable Credential or Verifiable Presentation are established, a <a>verifier</a> MUST validate included claims using their own business rules before relying on them. Such reliance SHOULD only occur after evaluating the issuer, the proof, the subject, and the claims, against verifier policy. 
```

Comment:
how do we test this MUST?

Reply:
It is not testable, nor does it need to be, imo.

How validation occurs is out of scope, but validating properties before using them is the only way this stuff works. @OR13 are you suggesting that verifiers use properties they don't understand?

Reply:
I agree with the language, I was mostly commenting because every MUST changes the scope of the test suites, and in this case, I think we would decline to test this one, because as you said, its not testable as written.... its essentially saying "You MUST... do some undefined thing ... before you rely on claims".

Reply:
While it is our commitment to test all testable normative requirements, this is a normative requirement, IMO, that is not testable, as all of the activity is downstream into and out of validation.

How validation is invoked is out of scope.
How validation is performed is out of scope.
How validation is reported is out of scope.
How verifiers handle the possible responses from validation is out of scope.

However, validation MUST occur before reliance on the property.

Validation is required to ensure the semantic layer of VCs, especially whether or not the issuer is an acceptable authority for a given claim and whether or not the cryptography is acceptable.

I think we would all be in agreement that, as a matter of principle, a verifier must not rely on properties for which they do not recognize the issuer as authoritative. 

That is the process of validation and, IMO, required. 

If its not normatively required, then we would be suggesting that verifiers can rely on properties just because they are signed and current--regardless of the issuer. That suggestion is clearly out of sync with the majority of use cases we have documented (where subject==holder) and basically the entire three-party model. One doesn't simply trust a VC because it verifiers, one must also make sure it is valid for the use case at hand. That is, verifiers MUST satisfy their own validation rules before relying on anything in a VC.

Given that fundamental aspect of VCs--that verifiers ALWAYS have to validate--it's clearly a MUST, despite the slight dissatisfaction from our inability to test that.

Reply:
The language is more clear without the RFC language. You just assert what you want to happen as fact, no need for RFC language here as it muddies the water from a testing perspective. We have to explain every normative statement that we're not testing, and this one would be hard to defend (and doesn't really add anything beyond just stating that verifiers validate stuff before they use it -- the ones that don't are bad verifiers and will get themselves into hot water (by not doing any validation) almost immediately).

Reply:
I added an issue #1328 to track the normative nature of validation.